hls4ml/backends/quartus/quartus_backend.py

```diff
@@ -115,35 +115,43 @@ def gen_quartus_weight_array(self, layer):
         layer.weights['weight'].data_length = layer.weights['weight'].data.size
         return
 
-    def build(self, model, synth=True, fpgasynth=False):
+    def build(self, model, synth=True, fpgasynth=False, logging=1, cont_if_large_area=False):
```

Comment:
Can we name it `log_level` or something more descriptive? Not to be confused that it is enabled/disabled with 0/1.

Reply:
Agreed.